docs/presigned_urls.md

```diff
@@ -0,0 +1,354 @@
+* All samples are of the same **cancer type** and have the same **assembly**
+
+&nbsp;
+### Configuration File Creation Scripts: Sample ID List
+
+The first step completed when [generate_config_files.py](../scripts/presigned_url_scripts/generate_config_files.py) is executed is the definition of the list the sample IDs, from the TSV file passed into the parameter `ids`. These sample IDs leverage the expected structure of the subdirectories within the S3 bucket (see the following section) to link appropriate files to their corresponding sample within the configuration file, as well as define the `id` property for each sample within the configuration file.
+
+The file used for this must be a tab-delimited file (TSV) that includes (but not limited to) a column named `ID`. The following example list, containing `n` samples:
+
+```tsv
+ID
+SAMPLE_1_ID
+SAMPLE_2_ID
+...
+SAMPLE_N_ID
+```
+will be converted to the following ID list:
+```
+[SAMPLE_1_ID, SAMPLE_2_ID,..., SAMPLE_N_ID]
+```
+Using this list, sample-wise configuration JSON objects are created iteratively, until all the objects are generated for all samples within the cohort. These IDs also correspond to subdirectories within the S3 bucket that contain that sample's files.
+
+?> Note: Samples are added to the configuration file in the order they are listed in the ID TSV file. To change the ordering of the samples within Chromoscope, the order can be manually altered within the ID file. Refer to the [following documentation](../scripts/clustering/README.md) for an explanation of hierarchical clustering using the Euclidean metric as a distance measure.
+
+&nbsp;
+### Configuration File Creation Scripts: S3 Bucket Subirectory Structure
+
+The configuration file creation script relies on an expected subdirectory structure within the S3 bucket, which is then accessed, sample-wise, according to the list of sample IDs from the submitted sample ID tsv file.
+
+If there are several cohorts within the S3 bucket containing the private data, each should have their own subdirectory:
+```bash
+EXAMPLE_S3_BUCKET/
+├── COHORT_1/
+├── COHORT_2/
+├── ...
+└── COHORT_N/
+```
+Otherwise, if there is only one cohort within the bucket, ignore the above.
+
+For the following level, the data should be subdivided into subdirectories corresponding to the following categories, with certain filetypes grouped together by sample (according the [data formats](data-formats.md) documentation):
+| Subdirectory | Number of files (per sample) | Filetypes | Required |
+|---|---|---|---|
+| Structural variants (SV) | 1 | `bedpe` | Yes |
+| Copy number variants (CNV) | 1 | `tsv` | Yes |
+| Driver mutations | 1 | `tsv`, `json` | No |
+| Point mutations (SNV) | 2 | `vcf` + `tbi` | No |
+| Insertion/deletion mutations (Indel) | 2 | `vcf` + `tbi` | No |
+| Read alignments | 2 | `bam` + `bai` | No |
+| Configuration files | - | `json` | No |
```

Comment:
This is broken on my local machine.

<img width="1856" alt="Screenshot 2023-04-24 at 17 41 20" src="https://user-images.githubusercontent.com/9922882/234122934-2695e587-7222-4f2a-94eb-8a41ec8dbc7c.png">